iotdb-protocol/thrift-datanode/src/main/thrift/datanode.thrift

```diff
@@ -804,7 +808,7 @@ service IDataNodeRPCService {
   /**
    * Config node will stop the Data node.
    */
-  common.TSStatus stopDataNode()
+  common.TSStatus stopDataNode(TStopDataNodeReq req)
```

Comment:
It appears that currently, this interface is only called when removing a datanode, and its parameter is always set to true. Can we assume this is a fixed behavior and directly proceed to delete the properties file? If, in the future, there’s a need to stop a node without deleting the properties file, we can then extend this interface. Additionally, its semantics now seem to be more than just stopping a node; perhaps it should be called stopAndClearDatanode?